pkg/apis/serving/v1/revision_types.go

```diff
@@ -157,6 +157,11 @@ type RevisionStatus struct {
 	// ref: http://bit.ly/image-digests
 	// +optional
 	ContainerStatuses []ContainerStatus `json:"containerStatuses,omitempty"`
+
+	// ActualReplicas reflects the PodAutoScalerStatus.ActualScale value in RevisionStatus.
+	ActualReplicas int32 `json:"actualReplicas,omitempty"`
+	// DesiredReplicas reflects the PodAutoScalerStatus.DesiredScale value in RevisionStatus.
```

Comment:
Assume user's/clients won't know what the PodAutoscaler is - so I'd change this comment to be explicit about what this number represents. ie. whether does it include terminating replicas etc..

Can we also add an the optional annotation to both these fields

```
// +optional
```